ui/DesignSystem/Primitive/Icon/Cross.svelte

```diff
@@ -1,50 +1,63 @@
 <script>
   export let style = null;
-  export let size = "small"; // normal | "big"
+  export let variant = "small"; // small | medium | big | filled | outline
```

Comment:
I'm actually in favor of not conflating `variant` and `size`; they're different things. A `variant` should refer to the UI element's visual state depending on its intended context (e.g. `filled`, `square`, `error`).

Reply:
I second that.

Reply:
I can see that. But then that would mean that you'd need to have different sizes for each variants. Which is not always the case.

Reply:
The way I would resolve this is by splitting out the "Filled in Cross" from the "Cross" component. Otherwise it seems messy.

Reply:
@sarahscott @rudolfs what do you think?

Reply:
I had another look at our icons and I think we'll have to put in a bit more thought to clean them up properly, but that is beyond the scope of this PR.

To me these things stand out:
  - we have two ways of specifying variants: via component name (`ArrowDown.svelte`, `ArrowUp.svelte`) and via the `variant` prop; ideally we settle on one way of doing things
  - we're mixing components of different complexities and use cases together: plain icons, and various animated spinners and transaction components.